pkg/kubelet/dockertools/manager.go

```diff
@@ -216,13 +218,21 @@ func NewDockerManager(
 		cpuCFSQuota:            cpuCFSQuota,
 	}
 	dm.runner = lifecycle.NewHandlerRunner(httpClient, dm, dm)
-	if serializeImagePulls {
-		dm.imagePuller = kubecontainer.NewSerializedImagePuller(kubecontainer.FilterEventRecorder(recorder), dm, imageBackOff)
-	} else {
-		dm.imagePuller = kubecontainer.NewImagePuller(kubecontainer.FilterEventRecorder(recorder), dm, imageBackOff)
-	}
 	dm.containerGC = NewContainerGC(client, containerLogsDir)
 
+	if dm.isDockerAPIVersionCompatible(minimumDockerAPIVersionForParallelPulls) {
```

Comment:
edit: shouldSerializeImagePulls doesn't need to take in the const minimumDockerAPIVersionForParallelPulls since it's a const :-)